packages/compass-aggregations/src/utils/stage.js

```diff
@@ -205,7 +204,10 @@ export const getStageHelpLink = (stageOperator) => {
   if (!stageOperator) {
     return null;
   }
-  return `${STAGE_HELP_BASE_URL}/${stageOperator.replace(/^\$/, '')}`;
+  return `https://www.mongodb.com/docs/manual/reference/operator/aggregation/${stageOperator.replace(
+    /^\$/,
+    ''
+  )}/?utm_source=compass&utm_medium=product`;
```

Comment:
Would this be valid for Cloud?

Reply:
Good point, we should double-check, but I'm pretty sure it's not. This is also relevant for other bits of compass used in DE: editor component, query bar. This also seems tricky to propagate this state in various places without making a mess out of it (like how do you tell compass-editor that it is in DE without passing some values all over the place?). Maybe we can find another way to make it work in Compass, maybe through the electron main process? We are in total control of opening urls after all 🤔

Reply:
Yeah I was thinking same. And we already have a [place to handle it](https://github.com/mongodb-js/compass/blob/main/packages/compass/src/main/window-manager.ts#L168) in `window-manager`.

Reply:
Perfect, that's probably the way to go about this, also just one place to apply the change (and no need to adjust urls)

Reply:
Ohh wow that is an interesting bit. Thank you @mabaasit @gribnoysup ⭐  I will not need any of the changes then, will close these PRs after I raise that.